assisted_compliance.md

```diff
@@ -0,0 +1,53 @@
+## Title
+
+Helping repo owners be compliant by writing their CONTRIBUTING.md for them as a pull request
+
+### Alternate titles
+
+* Let us help you be compliant
+* Assisted compliance
+
+## Problem
+
+The team that owns the repository doesn't have a CONTRIBUTING.md; the task force needs them to have this to submit bug fixes.
+
+## Context
+
+* Teams owning the repository are not complying with mandates for having a CONTRIBUTING.md
+* Compliance guys have to do a job; this is made difficult by teams resisting this.
+* Teams are focusing on business value; CONTRIBUTING.md compliance is viewed as not as important.
```

Comment:
```suggestion
* Teams owning the repository resist compliance-related mandates for having a CONTRIBUTING.md. Having this file is mandated in support of the InnerSource program, to ensure that there is a known and stated process for submitting PRs and having them be appropriately checked and accepted.
```

Fixing as suggested in https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/74/files#r169644654